### PR TITLE
nrunner: provide a Task identifier when none is given [v2]

### PR DESCRIPTION
**Pull request by clebergnu**
A task identifier is a mandatory attribute, and that fact is set clear in the class initializer. But, there are circumstances where users are not interested in manually set them, but just have them being unique.

---

Changes from v1 (#4461):
 * Changed the order of parameters in `Task`, making `identifier` the second (and optional) parameter

### CODE REVIEW

File: avocado/core/utils.py [hunk 1]
```diff
@@ -80,5 +79,5 @@ def resolutions_to_tasks(resolutions, config):
                                                  include_empty,
                                                  include_empty_key):
                     continue
-            tasks.append(Task(str(uuid1()), runnable, [status_server]))
+            tasks.append(Task(runnable, None, [status_server]))
```
Comment (richtja) [inline avocado/core/utils.py:82]: There is no need to have `None` here.

Comment (clebergnu) [reply avocado/core/utils.py:82]: Are you sure? The parameters are being passed based on their position.  If this parameter is omitted, then `[status_server]` is taken as an identifier.

Comment (richtja) [reply avocado/core/utils.py:82]: True, sorry that I didn't describe it correctly. I mean that we should use `Task(runnable,  status_uris=[status_server])`

Comment (richtja) [reply avocado/core/utils.py:82]: But that's just my preference

**Review by willianrampazzo — APPROVED** [review]
I'm also in favor of the @richtja proposal, but that is more related to coding style, so no major blocker here.

**Pull request merged**